0015-how-to-document-cnd-http-api.adoc

```diff
@@ -0,0 +1,207 @@
+= How to document the cnd HTTP API =
+Lucas Soriano del Pino <lucas@coblox.tech>;
+:toc:
+:revdate: 2019-08-29
+
+NOTE: Author: {authors} +
+Date: {revdate} +
+Tracking issue: https://github.com/comit-network/comit-rs/issues/1122[#1122]
+
+== Context ==
+
+We have no documentation for cnd's HTTP API. Having it is a hard requirement if we want to cater to application developers. It should also be helpful for the team and for newcomers.
+
+== Research ==
```

Comment:
nit: white space is non-uniform, sometimes a newline after a heading sometimes not.

Reply:
I think this doesn't matter unless you're looking at the raw file. Fixed it anyway :smile: